packages/database/src/migrations/20210621215753-LESMISDropNonOperationalSchools-modifies-data.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+import { SCHOOL_CODES } from './migrationData/20210621215753-LESMISDropNonOperationalSchools';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    -- Drop triggers so the large change notification doesn't slow us down
```

Comment:
I dropped these to speed up the migration but it still takes a few minutes, not sure if there's any cascading into other tables that I'm unaware of or just the fact that we're processing `1850` entries at once.

```
[INFO] Processed migration 20210621215753-LESMISDropNonOperationalSchools-modifies-data
Analytics refresh took: 3064ms
✨  Done in 184.95s.```

Reply:
a) should use disable/enable rather than drop/create triggers
b) it may be unsafe to not trigger the side effects of these changes...

Reply:
How long does it take if you leave triggers enabled?

Reply:
Took about an extra 90s, from discussion in person I've removed the drop triggers from this migration.